plugin/arcconf/arcconf.py

```diff
@@ -541,7 +539,7 @@ def disks(self, search_key=None, search_value=None,
                                     arcconf_disk, sys_id, cntrl_num))
 
         return search_property(rc_lsm_disks, search_key, search_value)
-
+    
```

Comment:
Please remove this white space.

Reply:
Done - commit id - 6651667aa6ba35ac90226cfc320baa488fd94e90